src/os/ui/config/betatogglesettingsui.js

```diff
@@ -0,0 +1,122 @@
+goog.declareModuleId('os.ui.config.BetaToggleSettingsUI');
+
+import AlertEventSeverity from '../../alert/alerteventseverity.js';
+import AlertManager from '../../alert/alertmanager.js';
+import Settings from '../../config/settings.js';
+import {ROOT} from '../../os.js';
+import Module from '../module.js';
+
+export const BETA_FEATURES_PATH = 'betaFeatures';
+
+/**
+ * The column mapping settings UI directive
+ * @return {angular.Directive}
+ */
+export const directive = () => ({
+  restrict: 'E',
+  replace: true,
+  templateUrl: ROOT + 'views/file/betatoggle.html',
+  controller: Controller,
+  controllerAs: 'ctrl'
+});
+
+/**
+ * The element tag for the directive.
+ * @type {string}
+ */
+export const directiveTag = 'betatoggle';
+
+
+Module.directive(directiveTag, [directive]);
+
+
+/**
+ * Controller for beta features settings.
+ * @unrestricted
+ */
+export class Controller {
+  /**
+   * Constructor.
+   * @param {!angular.Scope} $scope
+   * @ngInject
+   */
+  constructor($scope) {
+    /**
+     * @type {?angular.Scope}
+     * @protected
+     */
+    this.scope = $scope;
+
+    /**
+     * Settings instance for this class
+     * @type {Settings}
+     */
+    this.Settings = Settings.getInstance();
+
+    /**
+     * Array to hold all beta features
+     * @type {betaFeatureValues}
```

Comment:
this would technically be `Array<betaFeatureValues>` because it's an array that holds betaFeatureValues